dojo/tools/jfrog_xray_unified/parser.py

```diff
@@ -0,0 +1,118 @@
+        # not all cves have cvssv3 scores, so skip these. If no v3 scores, we'll default to index 0
+        if 'cvss_v3_score' in vulnerability['cves'][thisCveIndex]:
+            thisCvssV3Score = vulnerability['cves'][thisCveIndex]['cvss_v3_score']
+            if thisCvssV3Score > highestCvssV3Score:
+                highestCvssV3Index = thisCveIndex
+                highestCvssV3Score = thisCvssV3Score
+
+    # Following the CVSS Scoring per https://nvd.nist.gov/vuln-metrics/cvss
+    if 'severity' in vulnerability:
+        if vulnerability['severity'] == 'Unknown':
+            severity = "Info"
+        else:
+            severity = vulnerability['severity'].title()
+    # TODO: Needs UNKNOWN new status in the model.
+    else:
+        severity = "Info"
+
+    cveIndex = highestCvssV3Index
+
+    cve = "No CVE on file"
+    cvss_v3 = "No CVSS v3 score."
+    cvss_v2 = "No CVSS v2 score."
+    mitigation = "N/A"
+    extra_desc = ""
+
+    cves = vulnerability.get('cves', [])
+    if len(cves) > 0:
+        worstCve = cves[cveIndex]
+        if 'cve' in cves[cveIndex]:
+            cve = worstCve['cve']
+        if 'cvss_v3_vector' in worstCve:
+            cvss_v3 = worstCve['cvss_v3_vector']
+        if 'cvss_v2_vector' in worstCve:
+            cvss_v2 = worstCve['cvss_v2_vector']
+
+    if 'fixed_versions' in vulnerability and len(vulnerability['fixed_versions']) > 0:
+        mitigation = "Versions containing a fix:\n"
+        mitigation = mitigation + "</br>".join(vulnerability['fixed_versions'])
+
+    if 'external_advisory_source' in vulnerability and 'external_advisory_severity' in vulnerability:
+        extra_desc = vulnerability['external_advisory_source'] + ": " + vulnerability['external_advisory_severity']
+
+    if vulnerability['issue_id']:
+        title = vulnerability['issue_id'] + " - " + vulnerability['summary']
+    else:
+        title = vulnerability['summary']
+
+    references = ""
+    for reference in vulnerability['references']:
+        references += reference + "\n"
+
+    # component has several parts separated by colons. Last part is the version, everything else is the name
+    splitComponent = vulnerability['vulnerable_component'].split(':')
+    component_name = ":".join(splitComponent[:-1])
+    component_version = splitComponent[-1:][0]
+
+    # create the finding object
+    finding = Finding(
+        title=title,
+        cve=cve,
+        test=test,
+        severity=severity,
+        description=(vulnerability['description'] + "\n\n" + extra_desc).strip(),
+        mitigation=mitigation,
+        component_name=component_name,
+        component_version=component_version,
+        file_path=vulnerability['path'],
+        severity_justification="CVSS v3 base score: {}\nCVSS v2 base score: {}".format(cvss_v3, cvss_v2),
+        static_finding=True,
+        dynamic_finding=False,
+        references=references,
+        impact=severity,
+        cvssv3=cvss_v3)
```

Comment:
I think we can add more attribute.
like the `date` which seems to be the attribute `artifact_scan_time` in the report.
Also `"issue_id": "XRAY-139239"` should be the attribute `unique_id_from_tool` or `vuln_id_from_tool`

Reply:
fixed in https://github.com/DefectDojo/django-DefectDojo/pull/4551/commits/0f9f471c870e6ed15da7a17451c3ca986bc077a1